source/docs/zero-to-robot/step-1/how-to-wire-a-robot.rst

```diff
@@ -7,7 +7,7 @@ How to Wire an FRC Robot
 
    This document details the wiring of a basic electronics board for bench-top testing.
 
-   Some images shown in this section reflect the setup for a Robot Control System using Victor SPX Motor Controllers. Wiring diagram and layout should be similar for other motor controllers. Where appropriate, two sets of images are provided to show connections using controllers with and without integrated wires.
+   Some images shown in this section reflect the setup for a Robot Control System using SPARK or SPARK MAX Motor Controllers. Wiring diagram and layout should be similar for other motor controllers. Where appropriate, two sets of images are provided to show connections using controllers with and without integrated wires.
```

Comment:
Both should be listed- victor SPX is still used in some of the CTR images

Reply:
Besides the overall wiring diagram that shows multiple speed controllers and the overall robot picture, I didn't see any others. Did I miss one? I'm not really expecting people to wire off the full robot picture, and so it seems less confusing to not mention Victor SPX.